frontend/src/components/Collections/components/DeleteCollection/index.tsx

```diff
@@ -28,7 +28,7 @@ const DeleteCollection: FC<Props> = ({
 
   const handleDelete = async () => {
     setIsLoading(true);
-    await deleteMutation(id);
+    await deleteMutation({ collectionID: id });
     router.push(ROUTES.MY_COLLECTIONS);
```

Comment:
not sure if you care to move redirect into `{ onSuccess() { router.push(ROUTES.MY_COLLECTIONS); } }` here too